Jenkinsfile_CNP

```diff
@@ -10,8 +10,8 @@ def component = "manage-case-assignment"
 GradleBuilder builder = new GradleBuilder(this, product)
 
 // Variables to switch pipeline logic and wiring per type of build
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
-def dataStoreApiDevelopPr    = "PR-1260" // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-961"  // This doesn't change frequently, but when it does, only change this value.
```

Comment:
I guess we'll re-point to original ones once relevant PRs are merged, right?

Reply:
correct

Reply:
Fixed in https://github.com/hmcts/aac-manage-case-assignment/pull/294/commits/5c8856254497a08030df10133dd1957ee2f1e6c5